openmetadata-service/src/test/resources/openmetadata-secure-test.yaml

```diff
@@ -157,6 +157,20 @@ authenticationConfiguration:
     - "preferred_username"
     - "sub"
   enableSelfSignup : true
+  samlConfiguration:
```

Comment:
one check we need to do is either user can use sso, basic or samlConfiguration and throw an error if they configure multiple of them

Reply:
they can't actually, it's  controlled by provider